tests/spec/rainforest/joinByInvitation_logout_tryToLoginWith_NotAllowedEmail_InvalidPassword.rfml

```diff
@@ -7,7 +7,7 @@
 Click on the 'create a new team' link below the form where it says 'Welcome! Enter your team's Koding domain.'
 Do you see 'Let's sign you up!' form with 'Email Address' and 'Team Name' text fields and a 'NEXT' button?
 
-Enter "rainforest+{{ random.email }}" in the 'Email Address' and "{{ random.last_name }}{{ random.number }}" in 'Team Name' fields and click on 'NEXT' button
+Enter "{{ random.email }}" in the 'Email Address' and "{{ random.last_name }}{{ random.number }}" in 'Team Name' fields and click on 'NEXT' button
```

Comment:
`{{ random.email }}` doesn't solve the problems that `rainforest+{{ random.email }}` may cause. They have the same randomness. Also changing this value as `{{ random.email }}` breaks the next steps of the test where ``{{ random.email }}` is invited as a member (https://github.com/koding/koding/pull/9902/files?diff=split#diff-8f80098be99204fe0da8b9e9f3308253R25)